saleor/plugins/email_common.py

```diff
@@ -105,12 +115,54 @@ class EmailConfig:
 }
 
 
-def send_email(config: EmailConfig, recipient_list, template_name, context):
+def block_trans(this, options, *args, **kwargs):
+    # TODO add translation here
```

Comment:
Should we fix it before merge to the feature branch or it should be separate PR?

Reply:
Not in this PR. I need to figure out the best way for this.